.github/actions/slack-status/action.yml

```diff
@@ -0,0 +1,37 @@
+name: "Send Slack Status Notification"
```

Comment:
Could we move this to the workflows directory? I'd prefer to keep it as un-special as possible so all the tooling we're building around github actions doesn't need to be smart to handle this pattern

Reply:
So if I'm understanding this correctly, it's a composite action. Which differs from a reusable workflow. If we put it in the workflows, then it appears as it's own entity in your Actions tab in GH and would look like something you one can trigger. By making it an action, it basically acts like a single step in the update-helm-charts-index workflow.

If it's a concern, I think we'd probably be better off refactoring it out to a separate repository instead since we do appear to have a few copies floating around at this point.

Reply:
Thanks for looking into it! It would be nice to have it in a common repo somewhere, but I'm ok with leaving that for another PR.